src/satori.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import satori from 'satori'
+import { Resvg } from '@resvg/resvg-js';
+
+// failed
+// import { html } from "satori-html";
+
+// failed 
+const html = (...args: string[]) => {
+  // @ts-ignore
+  import('satori-html').then(({ html }) => html(...args));

```

Comment:
should be `return import('satori-html')`

Reply:
```ts
const html = (...args: string[]) => {
  // @ts-ignore
  return import('satori-html')
}```
?

Reply:
Tried all possible combinations `return import('..')` `return import('..').then...` `return import('..')(..args)`, they all give the same issue unless I'm missing something.

Reply:
like it was but with return.
`return import('satori-html').then(({ html }) => html(...args));`

Reply:
> they all give the same issue unless I'm missing something.

could you provide the issue?

Reply:
Same one
![image](https://user-images.githubusercontent.com/52013393/223077340-30cec003-842e-473f-997b-d6673201124b.png)

Reply:
`ts-node` compile dynamic import to `require` and it is cause the errors, you ahould configure ts to keep dynamic imports in the final bundle

Reply:
Changing `moduleResolution` to `node16` made your solution work! Thank you very much!